host-configs/rzvernal-toss_4_x86_64_ib_cray-clang@14.0.0_hip.cmake

```diff
@@ -92,7 +94,7 @@ set(MFEM_DIR "${TPL_ROOT}/mfem-4.4.0" CACHE PATH "")
 
 set(HDF5_DIR "${TPL_ROOT}/hdf5-1.8.22" CACHE PATH "")
 
-set(LUA_DIR "${TPL_ROOT}/lua-5.3.5" CACHE PATH "")
+set(LUA_DIR "${TPL_ROOT}/lua-5.4.4" CACHE PATH "")
```

Comment:
@white238 -- do we care about the LUA version and/or should we lock it down?

Reply:
There is no specific reason we are on one.